deps/wazuh_testing/wazuh_testing/scripts/simulate_agents.py

```diff
@@ -108,7 +108,10 @@ def create_agent(args, custom_labels):
         'syscollector_frequency': args.syscollector_frequency,
         'syscollector_event_types': args.syscollector_event_types,
         'syscollector_legacy_messages': args.syscollector_legacy_messages,
-        'syscollector_packages_vuln_content': args.syscollector_packages_list_file
+        'syscollector_packages_vuln_content': args.syscollector_packages_list_file,
+        'vulnerability_legacy_messages': args.vulnerability_legacy_messages,
+        'vulnerability_batch_size': args.vulnerability_batch_size,
```

Comment:
Could you clarify the concept of 'batch size' within the context of the vulnerability detection simulator? Currently, there should be only one 'osinfo' message and the remaining messages should pertain to packages. However, this implementation seems to deviate from the previous usage of 'syscollector'. Could you provide some insight into this change?

```suggestion
        'vulnerability_batch_size': args.vulnerability_batch_size,
```